docs/configuration.md

```diff
@@ -42,7 +42,7 @@ Some objects, such as: app-config, configmap-files, secret-files, dynamic-plugin
 
 #### Object annotation for mounting a volume to a specific path
 
-Using **rhdh.redhat.com/mount-path** annotation it is possible to define the directory where **PersistentVolumeClaim** object will be mounted to Backstage Container.
+Using **rhdh.redhat.com/mount-path** annotation it is possible to define the directory where **PersistentVolumeClaim** object will be mounted.
```

Comment:
The `pvcs.yaml` description in the table above should also be updated: https://github.com/redhat-developer/rhdh-operator/pull/582/files#diff-17ed18489a956f326ec0fe4040850c5bc9261d4631fb42da4c52891d74a59180L29